friends/src/Components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function Display(props) {
+    if (props.friends.length === 0) {
```

Comment:
// this check works perfectly fine but you could also accomplish the same thing like this
if (!props.friends) {
 ...
}